filesystems/s3/src/main/scala/cromwell/filesystems/s3/batch/S3BatchIoCommand.scala

```diff
@@ -56,11 +49,6 @@ import cromwell.filesystems.s3.S3Path
   * @tparam U Return type of the response
   */
 sealed trait S3BatchIoCommand[T, U] extends IoCommand[T] {
-  /**
-    * S3 operation to be executed by this command
-    */
-  def operation: S3Response
```

Comment:
The `operation` deletions are only semi-related. They weren't being used, which I discovered while tracing through some of the code